scripts/CEE/describe-kafka-transaction/metadata.yaml

```diff
@@ -19,9 +19,9 @@ rbac:
         - "get"
         - "list"
       apiGroups:
-        - "apps"
+        - "kafka.strimzi.io"
```

Comment:
services is part of the core api group of kube.  it is not part of strimzi. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#service-v1-core

Reply:
Change looks good to me